nginx/config/origin.go

```diff
@@ -28,8 +28,53 @@ server {
   gzip on;
   gzip_types text/plain test/csv application/json;
 
+  # Timeout configurations from origin server config
+  proxy_connect_timeout {{.backend_timeout}};
+  proxy_send_timeout {{.upload_timeout}};
+  proxy_read_timeout {{.download_timeout}};
+  
+  # Keepalive settings
```

Comment:
Can you describe what is meant by these keep alive setting you are adding and why are they being added ?